readme.txt

```diff
@@ -201,9 +200,9 @@ form triggers validation errors when UPE checkout is enabled.
 * Dev - Refactor the processing part of Webhook Controller to a separate service.
 * Dev - Remove type "Tweak" from the list of changelog types.
 * Dev - REST API documentation
+* Dev - Skip e2e tests if WC version is 5.0.0 because of WooCommerce Checkout Blocks minimum WC Required version
 * Dev - Unit test support for PHP 8 and upgrade PHPUnit version to 9.5.14
 * Dev - Updated contribution notes (how to add a changelog)
-* Dev - Skip e2e tests if WC version is 5.0.0 because of WooCommerce Checkout Blocks minimum WC Required version
```

Comment:
Changes in `readme.txt` are from cherry-picking 76dcc9691906a1160829da4d9673448915e09809.